config/routes.rb

```diff
@@ -14,6 +14,8 @@
   resources :carts, only:[:show]
 
   post '/orders/:id', to: 'orders#create'
+  post '/add_item', to: 'carts#add_item'
+  post '/subtract_item', to: 'carts#subtract_item'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.